lego/apps/events/views.py

```diff
@@ -97,13 +99,29 @@ def perform_update(self, serializer):
     def administrate(self, request, *args, **kwargs):
         event_id = self.kwargs.get('pk', None)
         queryset = Event.objects.filter(pk=event_id).prefetch_related(
-            'pools',
             'pools__permission_groups',
-            Prefetch('pools__registrations', queryset=Registration.objects.select_related('user')),
+            Prefetch('pools__registrations', queryset=Registration.objects.select_related(
+                'user'
+            ).prefetch_related('user__abakus_groups')),
             Prefetch('registrations', queryset=Registration.objects.select_related('user')),
         )
-        serializer = self.get_serializer(queryset.first())
-        return Response(serializer.data)
+        event = queryset.first()
+        event_data = self.get_serializer(event).data
+
+        grades = AbakusGroup.objects.filter(type=GROUP_GRADE)
```

Comment:
if you do `grades = AbakusGroup.objects.filter(type=GROUP_GRADE).values('id', 'name')` you'll get a list of dicts with `{'id: 1, 'name': 'webkom'}` (instead of using the serializer)

Reply:
Right, thats better 👍🏻